src/main/java/com/mael/ttt/Board.java

```diff
@@ -77,4 +77,12 @@ public int getIndexFromCoords(int row, int col) {
     public boolean isCellBusy(int index) {
         return !(board[getRowFromIndex(index)][getColFromIndex(index)] == EMPTY);
     }
+
+    public List<String> getValidIndexes() {
```

Comment:
"Valid" as in what sense?
Would it make sense here to actually only return the free locations on your board? Maybe that would simplify other checks.

What I find odd is, that you're returning a list of strings here, whereas `setCell()` takes an `int`. So there's a type mismatch in this class for me.